src/components/FooterRow.tsx

```diff
@@ -19,25 +19,35 @@ function Separator(): JSX.Element {
 
 /** Place me as a row in a container */
 export function FooterRow(): JSX.Element {
-  const clientContext = React.useContext(ClientContext);
+  const { client, nodeUrl, resetClient } = React.useContext(ClientContext);
 
   const [chainId, setChainId] = React.useState<string | ErrorState | LoadingState>(loadingState);
   const [height, setHeight] = React.useState<number | ErrorState | LoadingState>(loadingState);
 
-  const updateHeight = React.useCallback(() => {
-    clientContext.client
-      .getHeight()
-      .then(setHeight)
-      .catch(() => setHeight(errorState));
-  }, [clientContext.client]);
+  const updateHeight = React.useCallback(
+    client === null
+      ? () => {}
+      : () => {
+          client
+            .getHeight()
+            .then(setHeight)
+            .catch(() => setHeight(errorState));
+        },
```

Comment:
According to https://javascript.info/optional-chaining#short-circuiting, you should be able to do optional chaning with methods as well:

```ts
client?.getHeight() // …
```